### PR TITLE
Changed geography

### PR DESCRIPTION
**Pull request by garys-esri**
Moved from Afghanistan to California. This pull request will constitute a 4.1.0 release. This is a minor release, as opposed to a major release or a patch, for two reasons:

- Functionality added (new geography)
- Backward-compatible API changes (in military-apps-library-java)

### CODE REVIEW

**Comment by garys-esri** [comment]
@csmoore, if you could take a look and comment and/or merge at your convenience, I would appreciate it. Thanks in advance.

**Comment by csmoore** [comment]
Sure I will give a quick test within a few days.

File: source/VehicleCommander/mapconfig.xml [hunk 1]
```diff
@@ -39,21 +39,21 @@
         <!--layer name="Bihsud Bridge and Environs" visible="true" type="LocalDynamicMapLayer" basemap="false">
             <datasetpath>../../BihsudBridgeVCP.mpk</datasetpath>
         </layer> -->
-        <layer name="COA Sketch" visible="true" type="Mil2525CMessageLayer" basemap="false">
+        <!--layer name="COA Sketch" visible="true" type="Mil2525CMessageLayer" basemap="false">
```
Comment (csmoore) [inline source/VehicleCommander/mapconfig.xml]: This COA file was a nice way to test (especially control measures) - funnily enough the version installed in the Qt SDK is over San Francisco ArcGIS SDKs\Qt10.X\sdk\samples\data\symbology if you want to use that (I can provide if you don't have Qt SDK installed - I created that version by just moving the previous VC message file to SanFran)

File: license-ThirdParty.txt [hunk 2]
```diff
@@ -1,4 +1,4 @@
-This application includes FastVisibilityByDistance.gpk, a geoprocessing package (GPK) that contains elevation data from the Shuttle Radar Topography Mission (SRTM). See below for dataset citation details.
+This application includes FastVisibilityByDistance-MontereyCounty.gpk, a geoprocessing package (GPK) that contains elevation data from the Shuttle Radar Topography Mission (SRTM). See below for dataset citation details.
```
Comment (csmoore) [inline license-ThirdParty.txt]: Very, very minor "-" vs. "_"

Comment (garys-esri) [inline license-ThirdParty.txt]: Good catch. Fixed in 30985d0b018bec9e6e147802248616518c73d136 .

File: source/VehicleCommander/mapconfig.xml (hunk 1 above)
Comment (garys-esri) [inline source/VehicleCommander/mapconfig.xml]: We're focusing on Monterey, not San Francisco, so I'll make a new .xml file. It shouldn't be too hard (famous last words) using the Military Planner app.

Comment (garys-esri) [inline source/VehicleCommander/mapconfig.xml]: Fixed in e4dea03eba7c84b76c90703e50d9fcb0de14d1ec

**Comment by garys-esri** [comment]
@csmoore, I addressed the notes you entered. Please merge at your convenience unless you have found other things that need fixing.

File: source/VehicleCommander/afmchemlight.json [hunk 3]
```diff
@@ -1,124 +0,0 @@
-{
```
Comment (csmoore) [inline source/VehicleCommander/afmchemlight.json]: If we are deleting these custom json files, we should also update(remove) those copy steps in the readme:
https://github.com/Esri/vehicle-commander-java/blob/master/README.md#configurecopy-the-required-files

Comment (garys-esri) [inline source/VehicleCommander/afmchemlight.json]: You're right. That whole section in README.md is now irrelevant. I removed it in ac83f4a0af7d36b44ea071bb428160335e9eb418 .

**Comment by csmoore** [comment]
I like it:
image

Question: is there a geomessage simulator file somewhere I can use to test the tracks with the simulator?

**Comment by garys-esri** [comment]
Yes. See https://github.com/garys-esri/geomessage-simulator-qt/tree/master/GeoMessageSpecification/sample-messages .

**Comment by csmoore** [comment]
Thanks - everything looks good in a quick test.

image

**Comment by garys-esri** [comment]
Thanks!

**Comment by jrweakland** [comment]
:+1:

**Pull request merged**